model/src/main/proto/analytics.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Supported event actions for analytics logging
+enum EventAction{
+  // The default user action of an event
+  UNKNOWN_EVENT_ACTION = 0;
+}
+
+// Supported priority of events for analytics logging
+enum Priority{
+  // The default priority of an event
+  UNKNOWN_PRIORITY = 0;
+  // The priority of events whose logs should not be removed from the storage
```

Comment:
For the most part. We would still remove these logs if they are the only ones available right. So mention that

Reply:
Done.